<launch>
  <!-- 加载参数 -->
  <rosparam file="$(find my_robot_vision)/config/vision_params.yaml" command="load" />

  <!-- 启动服务节点 -->
  <node name="align_capture_service_node"
        pkg="my_robot_vision"
        type="align_capture_service_node.py"
        output="screen" />
</launch>
